src/components/typography/typography.tsx

```diff
@@ -44,47 +44,49 @@ export interface TypographyProps extends HTMLAttributes<HTMLElement> {
   intent?: Intent
 }
 
-const propsToClass = ({
-  className,
-  hasGreyText,
-  hasItalicText,
-  hasBoldText,
-  hasRegularText,
-  hasMediumText,
-  hasNoMargin,
-  hasMargin,
-  hasSectionMargin,
-  hasCenteredText,
-  hasDisabledText,
-  hasCapitalisedText,
-  hasUpperCasedText,
-  intent,
-  children,
-  ...rest
-}: TypographyProps) => () => {
-  const intentClass = intent ? getIntentClassName(intent) : null
-
-  return {
-    className: cx(
-      className,
-      hasGreyText && elHasGreyText,
-      hasRegularText && elHasRegularText,
-      hasBoldText && elHasBoldText,
-      hasItalicText && elHasItalicText,
-      hasMediumText && elHasMediumText,
-      hasMargin && elHasMargin,
-      hasNoMargin && elHasNoMargin,
-      hasSectionMargin && elHasSectionMargin,
-      hasCenteredText && elHasCenteredText,
-      hasDisabledText && elHasDisabledText,
-      hasCapitalisedText && elHasCapitalisedText,
-      hasUpperCasedText && elHasUpperCasedText,
-      intentClass,
-    ),
+const propsToClass =
```

Comment:
Can you run `yarn lint --fix` over the project please? There's some weird formatting going on

Reply:
Done. Made no change so will sort manually